### PR TITLE
Card Content Description - Fetch on mount

### PR DESCRIPTION
**Pull request by RohitRajendran**
### Description
Fetches data on mount in card content description

### Testing Instructions
1. If valid, should check on mount

### CODE REVIEW

File: components/molecules/CardContentDescription/CardContentDescription.js [hunk 1]
```diff
@@ -33,26 +34,42 @@ class CardContentDescription extends Component {
...
-          valid,
-        });
-
-        this.fetchData();
-      } else if (!valid && valid !== this.state.valid) {
-        // If the field is invalid then the following state causes the text to hide.
-        this.setState({
-          valid,
-        });
+      if (values !== prevValues) {
```
Comment (CalMlynarczyk) [inline components/molecules/CardContentDescription/CardContentDescription.js]: The validation function won't fire now unless you change another one of the specified values. I think we have to always run the validation function like it was doing before. There is no guarantee that the validation function can't change without the `values` prop changing.

Comment (RohitRajendran) [reply components/molecules/CardContentDescription/CardContentDescription.js]: I still run the validation function but now I just check in the checkValidityAndFetch function

Comment (RohitRajendran) [reply components/molecules/CardContentDescription/CardContentDescription.js]: Ohhh I get what you mean

Comment (RohitRajendran) [reply components/molecules/CardContentDescription/CardContentDescription.js]: Fixed

**Pull request merged**